fuel-client/src/client/schema/tx/tests/transparent_tx.rs

```diff
@@ -242,26 +241,37 @@ impl TryFrom<Input> for fuel_tx::Input {
                 contract_id: contract.contract.id.into(),
             },
             Input::InputMessage(message) => {
+                let data_len = message.data.0 .0.len();
+
+                let recipient = {
```

Comment:
There shouldn't be a need for the client to get what was once referred to as "recipient" (the target fuel contract to send the message to). `fuel_tx::Input::MessageSigned` shouldn't have an owner field anymore and the recipient in this case should just be `recipient: message.recipient.into(),` and data should be left like it was before `data: message.data.into()`

Reply:
I see, it makes sense, it would've been helpful it this issue was tagged as blocked by `fuel-tx` issue, but I can fix it fast once that is done